source/DasBlog.Web.UI/Startup.cs

```diff
@@ -288,7 +288,8 @@ public void ConfigureServices(IServiceCollection services)
 			{
 				options.WaitForJobsToComplete = true;
 			});
-		}
+            services.AddApplicationInsightsTelemetry(Configuration["APPINSIGHTS_INSTRUMENTATIONKEY"]);
+        }
```

Comment:
Does this get stored in appsettings.json?

Reply:
Yes, or in your AppService ENV variables. I believe it's optional.